sds-go/go/go.mod

```diff
@@ -1,3 +1,3 @@
-module sds
+module github.com/DataDog/dd-sensitive-data-scanner/sds-go/go
```

Comment:
@remeh just for my understanding what does it change for external repo that would use this ?

Reply:
Yes I got that but what does it change for outside if you let module sds instead ?

Reply:
It can't be imported/resolved as `sds` is not the full address of the pkg.

A random example of another pkg that is resolvable from the outside, from another repo if that helps: https://github.com/DataDog/datadog-agent/blob/main/pkg/tagger/go.mod

Reply:
ok thks